requirements/extras/redis.txt

```diff
@@ -1 +1 @@
-redis>=4.2.2,<4.4.0
+redis>=4.3.6,<4.6.0
```

Comment:
I think you now want 4.3.7.  GitHub pipelines seem plugged - but it should get there as soon as things wake up.

Reply:
yes but I haven't seen any release for that yet https://github.com/redis/redis-py/releases